assignments/assignment2/fileresult.txt

```diff
@@ -0,0 +1,15 @@
+Output for command "aarch64-none-linux-gnu-gcc -print-sysroot"
```

Comment:
@aasu8675 looks like you are missing the output of the `file` command and have written the cross-compile content here instead.